permuta/MeshPatt.py

```diff
@@ -272,16 +265,16 @@ def shade(self, positions):
 
         return MeshPatt(self.pattern, self.shading | positions)
 
-    def add_point(self, box, shade_dir=DIR_NONE, safe=True):
-        """Returns a mesh pattern with a point added in box.  If shade_dir is
+    def add_point(self, pos, shade_dir=DIR_NONE, safe=True):
```

Comment:
There is a bunch of stuff that we should refactor, including variable names. Function names seem to be mostly ok, but we might also want to reconsider the format of the docstrings.